components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -513,7 +515,7 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
                 _lockConfig.Release();
             }
 
-            return new List<string> { "Succeeded" };
+            return response;
```

Comment:
do we want to send an empty list here?

Reply:
We add the message in line above for successful publishing for the given endpoint URL.

Reply:
oh, I missed that, I thought it's a log entry. I'm not sure though if that response brings any useful information for the caller. I mean when the client calls the direct method, the DM's name is already known and so is the data set delta's  endpoint url. I think if we cannot add more than superfluous info in the response, we'd better leave it empty,